katalog/istio/README.md

```diff
@@ -17,169 +34,175 @@ Source: [https://istio.io/docs](https://istio.io/docs/concepts/what-is-istio/#wh
-  if the initial call fails.
-  - **Circuit breakers:** In a circuit breaker, you set limits for calls to individual hosts within a service, such
-  as the number of concurrent connections or how many times calls to this host have failed.
-  - **Fault injection:** Fault injection is a testing method that introduces errors into a system to ensure that it
-  can withstand and recover from error conditions.
-- ***Policies:*** Istio lets you configure custom policies for your application to enforce rules at runtime such as:
-  - Rate limiting to dynamically limit the traffic to a service
-  - Denials, whitelists, and blacklists, to restrict access to services
-  - Header rewrites and redirects
-- ***Observability:*** Istio generates detailed telemetry for all service communications within a mesh. This telemetry
-provides observability of service behavior, empowering operators to troubleshoot, maintain, and optimize their
-applications – without imposing any additional burdens on service developers.
-
-## Image repository and tag
-
-All istio container images are currently available at dockerhub: [docker.io/istio](https://hub.docker.com/u/istio)
-
-* istio container images: `docker.io/istio/*`
-
-Includes:
-
-- docker.io/istio/proxyv2
-- docker.io/istio/mixer
-- docker.io/istio/pilot
-
-* addons container images:
-
-- Kiali: quay.io/kiali/kiali:v1.9
-
-## Deployment
-
-You can deploy istio by running following command in the root of the project:
-
-```shell
-$ kustomize build | kubectl apply -f -
+In order to install the Istio package, it's required to have installed the [init](./init/README.md) package.
+This is the one involved in the Istio CRDs creation.
+
+```bash
+$ kustomize build katalog/istio/init | kubectl apply -f -
+$ kubectl -n istio-system wait --for=condition=complete job --all
+job.batch/istio-init-crd-10-1.4.2 condition met
+job.batch/istio-init-crd-11-1.4.2 condition met
+job.batch/istio-init-crd-14-1.4.2 condition met
 ```
 
-### Configuration
+Once installed, the cluster will be ready to install the [minimal](#minimal-installation) Istio deployment.
 
-#### Kiali
 
-##### Admin user
+## Modular
 
-We provide an administrator user *(admin:admin)* to access the kiali dashboard. Please change the default password
-editing the `kiali` secret inside the `istio-system` namespace.
+Istio has a lot of features powered by a high number of pieces. This package has been designed to be installed from the
+minimal installation providing basic features like traffic routing to a more complex deployment including
+more features.
 
-```bash
-$ kubectl get secret kiali -o yaml -n istio-system
-apiVersion: v1
-kind: Secret
-metadata:
-  name: kiali
-  namespace: istio-system
-  labels:
-    app: kiali
-type: Opaque
-data:
-  passphrase: YWRtaW4=
-  username: YWRtaW4=
+
+### Minimal installation
+
+The [minimal](minimal/README.md) deployment is a perfect starting point to play with Istio. It deploys the basic
```

Comment:
- Is `minimal` a deployment, a module or a package? The three concepts are used along the readme interchangeably and it's confusing.
- I would replace "play" with something like "get familiar with", it's more professional:
"The [minimal](minimal/README.md) deployment is a perfect starting point to get familiar with Istio."

Reply:
About the nomenclature... you're right. Sometimes i mix concepts :(. The minimal istio deployment is a package. 

Agree changing play -> get familiar with. Changing!